Woof/Woof/Model/Preferences/PreferencesHandler.swift

```diff
@@ -25,7 +25,7 @@ enum PreferencesHandler {
 
      - Returns: The selected role for the user in the app session. If the role couldn't be loaded, returns `.none`.
      */
-    static func getUserRole() -> Role {
+    @discardableResult static func getUserRole() -> Role {
```

Comment:
For me it doesn't make sense to do this method with `discardableResult`.
Could you explain the reason to do that?

Reply:
to avoid warning in this test
``` swift
    func testExistsInAPI() {
        PreferencesHandler.getUserRole()
    }
```

Reply:
you can call this method like this in tests
```swift
func testExistsInAPI() {
        _ = PreferencesHandler.getUserRole()
    }
```

Reply:
it's a bad idea to use `discardableResult` for this method because we remove warnings in other places that may appear?

Reply:
Fixed